app/javascript/components/home/HomePage.jsx

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import { Card, Form, Stack } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
 import ButtonLink from '../shared_components/utilities/ButtonLink';
 import Spinner from '../shared_components/utilities/Spinner';
 import useEnv from '../../hooks/queries/env/useEnv';
 import Logo from '../shared_components/Logo';
+import { useAuth } from '../../contexts/auth/AuthProvider';
 
 export default function HomePage() {
   const { isLoading, data: env } = useEnv();
+  const currentUser = useAuth();
+
+  function homepageButtons() {
+    if (currentUser?.signed_in) {
+      return (
+        <div className="text-start">
+          <h6> Please enter the URL of your meeting. </h6>
+          <Stack direction="horizontal">
+            <input className="form-control" id="joinUrl" />
+            <Button variant="brand" className="ms-2">Join</Button>
+          </Stack>
+        </div>
+      );
+    }
```

Comment:
These are 2 different pages - I don't believe we should force them to work together